### PR TITLE
Add output path to react-components preset

### PR DESCRIPTION
**Pull request by helfi92**

### CODE REVIEW

File: packages/react-components/index.js [hunk 1]
```diff
@@ -68,6 +68,7 @@ module.exports = (neutrino, opts = {}) => {
           .hints('error')
           .end()
         .output
+          .path(neutrino.options.output)
```
Comment (eliperelman) [inline packages/react-components/index.js]: Isn't this caught by the web preset?

Comment (eliperelman) [reply packages/react-components/index.js]: https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/web/index.js#L134

Comment (helfi92) [reply packages/react-components/index.js]: Yes, that was a silly PR. Closing this now :)

Comment (eliperelman) [reply packages/react-components/index.js]: lol no worries!

**Pull request closed without merging**